examples/language-modeling/scripts/Qwen1.5-7B-Chat.sh

```diff
@@ -10,3 +10,11 @@ python3 main.py \
 --sym \
 --minmax_lr 0.01 
 
+# Evaluation command for CEVAL, CMMLU, MMLU and GSM8K tasks
+lm_eval --model hf \
+    --model_args pretrained="gpu_model_path",autogptq=True,gptq_use_triton=True,trust_remote_code=True \
+    --device cuda:0 --tasks ceval-valid,cmmlu,mmlu --batch_size 16
```

Comment:
the default num_few_shot should be zero, move the cmd to acc.md

Reply:
the default num_fewshot for gsm8k is 5 and 0 for the rest tasks. 
It's been moved to acc.md